packages/experimental/Badge/src/Badge.types.ts

```diff
@@ -0,0 +1,134 @@
+import { ViewStyle, ColorValue } from 'react-native';
+import { TextProps } from '@fluentui-react-native/experimental-text';
+import { FontTokens, IBorderTokens, IColorTokens, IShadowTokens, LayoutTokens } from '@fluentui-react-native/tokens';
+import type { IViewWin32Props } from '@office-iss/react-native-win32';
```

Comment:
This should use ViewProps - otherwise the Badge is restricted to win32. My understanding is that our adapter package should do the right thing here and use the right set of props for each platform. Do we need the win32 version specifically for any reason?

Reply:
I think this should actually be `IViewProps` coming from `@fluentui-react-native/adapters`, so that adapters can do the right thing where this is referenced.

Reply:
I don't think it should be only for win32. Sorry, I missed that part. It was in PersonaCoin component.